src/cpp/session/resources/templates/shiny.qmd

```diff
@@ -17,6 +17,8 @@ plotOutput("distPlot")
 output$distPlot <- renderPlot({
    x <- faithful[, 2]  # Old Faithful Geyser data
    bins <- seq(min(x), max(x), length.out = input$bins + 1)
-   hist(x, breaks = bins, col = 'darkgray', border = 'white')
+   hist(x, breaks = bins, col = 'darkgray', border = 'white',
+        xlab = "Waiting time to next eruption (in mins)",
```

Comment:
Nit: it looks like we're using single quotes above, and double quotes here. Can we change these to be consistent?

I think the tidyverse style guide recommends using double quotes, so we could also just change the other usages in these documents to be double quotes as well.